laundry/api_wrapper.py

```diff
@@ -0,0 +1,229 @@
+import os
+
+import requests
+from bs4 import BeautifulSoup
+from django.core.management import call_command
+from django.db.utils import OperationalError, ProgrammingError
+from django.utils import timezone
+
+from laundry.models import LaundryRoom, LaundrySnapshot
+
+
+LAUNDRY_URL = os.environ.get("LAUNDRY_URL", "http://suds.kite.upenn.edu")
+ALL_URL = f"{LAUNDRY_URL}/?location="
+
+# USAGE_BASE_URL from LAS does not work
+
+
+class Laundry(object):
+    def __init__(self):
+        self.busy_dict = {
+            "LowBusyNightColor": "Low",
+            "LowBusyDayColor": "Low",
+            "MediumLowBusyColor": "Medium",
+            "MediumHighBusyColor": "High",
+            "HighBusyColor": "Very High",
+            "NoDataBusyColor": "No Data",
+        }
+        self.days = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"]
+        self.hall_to_link = {}
+        self.id_to_hall = {}
+        self.id_to_location = {}
+        self.hall_id_list = []
+        self.create_hall_to_link_mapping()
+
+    def create_hall_to_link_mapping(self):
+        try:
+            # makes migration when there are no rooms
+            if LaundryRoom.objects.count() == 0:
```

Comment:
I don't think we should have this code. Since the command only needs to be run once, then the db will always have the info